Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertFaceWall.cs

```diff
@@ -60,16 +60,16 @@ public List<ApplicationPlaceholderObject> FaceWallToNative(RevitFaceWall speckle
       // otherwise the wall creation will fail with "Could not create a face wall."
       var level = new FilteredElementCollector(Doc)
          .WhereElementIsNotElementType()
-         .OfCategory(BuiltInCategory.OST_Levels)
-         .ToElements().First();
+         .OfCategory(BuiltInCategory.OST_Levels) // this throws a null error if user tries to recieve stream in a file with no levels
+         .ToElements().FirstOrDefault();
```

Comment:
I think you're missing the level null check and the creation of the new one here?